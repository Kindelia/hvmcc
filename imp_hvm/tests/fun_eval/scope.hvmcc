function Main () {
  {Main} =
    let pair = {Pair 2 3};
    let _  = match pair {
      {Pair x y} =>
        let pair = {Pair 3 4};
        pair
    };
    pair
}
