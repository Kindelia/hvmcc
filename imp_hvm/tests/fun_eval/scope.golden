{Pair 2 3}